server/service/src/stocktake/update/generate.rs

```diff
@@ -76,9 +76,9 @@ fn generate_stock_in_out_or_update(
     stocktake_line: &StocktakeLine,
     stock_line: &StockLineRow,
 ) -> Result<StockLineJob, UpdateStocktakeError> {
-    let row = stocktake_line.line.to_owned();
+    let stocktake_line = stocktake_line.line.to_owned();
```

Comment:
nit: consistency would be to call this stocktake_line_row like you did in `generate_new_stock_line`